test/encoding_test.dart

```diff
@@ -553,8 +553,8 @@ void main() {
           Box(Point(1, -1), Point(-1, 1)),
           Box(Point(1, 1), Point(-1, -1)),
           Box(Point(-1, -1), Point(1, 1)),
-          Box(Point(0,0),Point(-1,0)),
-          Box(Point(0,0),Point(0,0)),
+          Box(Point(0, 0), Point(-1, 0)),
+          Box(Point(0, 0), Point(0, 0)),
```

Comment:
Are we not using `dart format` in the CI :scream:?

Reply:
🙈